sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -759,6 +772,7 @@ def check(self, _):
                 self.procedure_metrics.run_job_loop(self.tags)
                 self.activity.run_job_loop(self.tags)
                 self.sql_metadata.run_job_loop(self.tags)
+                self._schemas.run_job_loop(self.tags)
```

Comment:
if we are running in this a separate thread do we still need a `max_execution_time` parameter? OR, if we want to keep it, can it at least be much greater than 10 seconds by default? If the collection interval is 600 seconds it doesn't make much sense for this to be so low

Reply:
I don't plan on including it in this change. It's an interesting idea that we can consider but we have other improvements which may eliminate the need for a schema thread. 

I added the idea of a async to this ticket though
https://datadoghq.atlassian.net/browse/DBMON-4150